gammapy/analysis/config.py

```diff
@@ -229,6 +229,7 @@ class LogConfig(GammapyBaseConfig):
 class GeneralConfig(GammapyBaseConfig):
     log: LogConfig = LogConfig()
     outdir: str = "."
+    n_jobs: int = 1
```

Comment:
For now `n_jobs` is used mostly in data reduction, but could be used in later steps so this is likely a valid place to put it. Is that the motivation?

Reply:
yes, I see other potential use cases:
- TSmapEstimator already have this argument,
- the ObservationsEventsSampler from #3711 will,
- run multiple flux points in parallel from a pool of sources and energy bins,
- allow multiple kernel size/shape for the ExcessMapEstimator/TSmapEstimator and run them in parallel
Moreover even if later we want to define a n_jobs per analysis step, I think that we still need to define a general limit at the beginning.